Package.swift

```diff
@@ -0,0 +1,16 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SwiftTickerView",
+    products: [
```

Comment:
Could you pls add 
platforms: [
        .iOS(.v9),
        .tvOS(.v9)
    ],
here?